src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ThreadPoolTaskScheduler.cs

```diff
@@ -42,16 +42,16 @@ internal ThreadPoolTaskScheduler()
         protected internal override void QueueTask(Task task)
         {
             TaskCreationOptions options = task.Options;
+#if !TARGET_BROWSER
```

Comment:
`Thread.IsThreadStartSupported` condition should take care of stripping this for Browser. This ifdef should not be needed.

Reply:
I wasn't sure whether it was needed or not; I was just moving it because where it was previously made little sense :)

I can remove it entirely though if it's safe to do so.